configs/tridentnet/README.md

```diff
@@ -1,7 +1,6 @@
 # Scale-Aware Trident Networks for Object Detection
 
 ## Introduction
-
```

Comment:
Could you also add a tag here as `[ALGORITHM]` as that in [here](https://github.com/open-mmlab/mmdetection/tree/master/configs/cascade_rcnn).